apps/src/javalab/Neighborhood.js

```diff
@@ -0,0 +1,35 @@
+const maze = require('@code-dot-org/maze');
+const MazeController = maze.MazeController;
+var tiles = require('@code-dot-org/maze').tiles;
+var Direction = tiles.Direction;
+
+export const afterInject = (level, skin, config, studioApp) => {
+  // Insert some temporary values here until we can populate them from levelbuilder
+  level.map = [
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 2, 1, 1, 3, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0]
+  ];
+  level.startDirection = Direction.EAST;
+
+  let controller = new MazeController(level, skin, config, {
+    methods: {
+      playAudio: (sound, options) => {
+        studioApp.playAudio(sound, {...options, noOverlap: true});
+      },
+      playAudioOnFailure: studioApp.playAudioOnFailure.bind(studioApp),
+      loadAudio: studioApp.loadAudio.bind(studioApp),
+      getTestResults: studioApp.getTestResults.bind(studioApp)
+    }
+  });
+  const svg = document.getElementById('svgMaze');
```

Comment:
where does 'svgMaze' come from?

Reply:
Unfortunately, it's a magic value that gets set up by the Maze Visualization area and gets used in the maze repo and the code-dot-org repo. I'll add a comment.

Reply:
✔️